templates/helm/templates/_helpers.tpl

```diff
@@ -30,3 +30,9 @@ If release name contains chart name it will be used as a full name.
 {{- define "service-account.name" -}}
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
+
+{{- define "watch-namespace" -}}
+{{- if eq .Values.installScope "namespace" -}}
+{{- .Release.Namespace -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Have we confirmed that this defaults to an empty string when install scope is cluster?

Reply:
```
Environment:
      K8S_NAMESPACE:                   ack-system (v1:metadata.namespace)
      AWS_ACCOUNT_ID:                  123456789012
      AWS_REGION:                      us-west-1
      AWS_ENDPOINT_URL:                
      ACK_WATCH_NAMESPACE:             
      ACK_ENABLE_DEVELOPMENT_LOGGING:  true
      ACK_LOG_LEVEL:                   debug
      ACK_RESOURCE_TAGS:               services.k8s.aws/managed=true,services.k8s.aws/created=%UTCNOW%,services.k8s.aws/namespace=%KUBERNETES_NAMESPACE%
      AWS_ROLE_ARN:                    arn:aws:iam::123456789012:role/ack-controller-role-namespaced
      AWS_WEB_IDENTITY_TOKEN_FILE:     /var/run/secrets/eks.amazonaws.com/serviceaccount/token
```

Defaults to an empty string when using cluster.
When using namespace, value is correctly passed.

```
  Environment:
      K8S_NAMESPACE:                   ack-system (v1:metadata.namespace)
      AWS_ACCOUNT_ID:                  123456789012
      AWS_REGION:                      us-west-1
      AWS_ENDPOINT_URL:                
      ACK_WATCH_NAMESPACE:             ack-system
      ACK_ENABLE_DEVELOPMENT_LOGGING:  true
      ACK_LOG_LEVEL:                   debug
      ACK_RESOURCE_TAGS:               services.k8s.aws/managed=true,services.k8s.aws/created=%UTCNOW%,services.k8s.aws/namespace=%KUBERNETES_NAMESPACE%
      AWS_ROLE_ARN:                    arn:aws:iam::123456789012:role/ack-controller-role-namespaced
      AWS_WEB_IDENTITY_TOKEN_FILE:     /var/run/secrets/eks.amazonaws.com/serviceaccount/token

```